### PR TITLE
Extend countries spoke view from hub view

### PR DESCRIPTION
**Pull request by ANich**

### CODE REVIEW

File: mozilla_vpn/views/funnel_product_page_to_subscribed_table.view.lkml [hunk 1]
```diff
@@ -147,18 +147,19 @@ view: +funnel_product_page_to_subscribed_table {
...
+  #   type: sum
+  # }
+  # measure: subscribe_coupon_fail {
+  #   sql: ${TABLE}.subscribe_coupon_fail ;;
+  #   type: sum
+  # }
+  # measure: subscribe_coupon_success {
+  #   sql: ${TABLE}.subscribe_coupon_success ;;
+  #   type: sum
+  # }
```
Comment (ANich) [inline mozilla_vpn/views/funnel_product_page_to_subscribed_table.view.lkml:162]: cc @relud @YeonjooSmith seems like there's a redefinition error in this view - [example](https://mozilla.cloud.looker.com/explore/mozilla_vpn/funnel_product_page_to_subscribed_table?):
```
Redefinition of field 'funnel_product_page_to_subscribed_table.subscribe_coupon_submit'
[spoke-default/mozilla_vpn/views/funnel_product_page_to_subscribed_table.view.lkml:150](https://mozilla.cloud.looker.com/projects/spoke-default/files/mozilla_vpn/views/funnel_product_page_to_subscribed_table.view.lkml?line=150)
```

Comment (YeonjooSmith) [reply mozilla_vpn/views/funnel_product_page_to_subscribed_table.view.lkml:162]: @ANich 
The error showed up then it disappeared after refreshing. I looked up [the redefinition error](https://help.looker.com/hc/en-us/articles/360017132614-Error-Redefinition-of-Field-) and I double checked if there are duplicates in dimensions/measures. Let me know if the issue has not been resolved.

Comment (YeonjooSmith) [reply mozilla_vpn/views/funnel_product_page_to_subscribed_table.view.lkml:162]: Ohh I see! I didn't know that the view in looker-hub was automatically updated. I renamed my measures :) 
BTW, when I commit those changes, I get errors (from clicking `validate LookML`) from the explores from Looker-Hub. In this case, what should I do to commit my changes without having the permission to modify files in Looker-Hub?

`LookML Errors (3)
There are serious errors with this LookML code that could prevent queries from running.
Unknown or inaccessible field "metrics.metrics__labeled_counter__recent_synced_tabs_recent_synced_tab_opened" referenced in "metrics__metrics__labeled_counter__recent_synced_tabs_recent_synced_tab_opened". Check for typos or append a timeframe to the name if the field is type "time".
[metrics.explore:247 fenix:metrics](https://mozilla.cloud.looker.com/projects/looker-hub/files/fenix/explores/metrics.explore.lkml?line=247)[](https://looker.com/docs/r/develop/unknown-inaccessible-field/578?version=22.4)
Unknown or inaccessible field "metrics.metrics__labeled_counter__recent_synced_tabs_recent_synced_tab_shown" referenced in "metrics__metrics__labeled_counter__recent_synced_tabs_recent_synced_tab_shown". Check for typos or append a timeframe to the name if the field is type "time".
[metrics.explore:252 fenix:metrics](https://mozilla.cloud.looker.com/projects/looker-hub/files/fenix/explores/metrics.explore.lkml?line=252)[](https://looker.com/docs/r/develop/unknown-inaccessible-field/578?version=22.4)
Unknown or inaccessible field "metrics.metrics__labeled_counter__tabs_tray_access_point" referenced in "metrics__metrics__labeled_counter__tabs_tray_access_point". Check for typos or append a timeframe to the name if the field is type "time".
[metrics.explore:257 fenix:metrics](https://mozilla.cloud.looker.com/projects/looker-hub/files/fenix/explores/metrics.explore.lkml?line=257)`

Comment (ANich) [reply mozilla_vpn/views/funnel_product_page_to_subscribed_table.view.lkml:162]: Uncovered an issue looking into this haha https://github.com/mozilla/lookml-generator/issues/412 but Anna implemented a quick workaround here https://github.com/mozilla/looker-spoke-default/pull/300 (and renamed the fields as well)

**Pull request closed without merging**